py-polars/polars/utils/udfs.py

```diff
@@ -354,7 +368,7 @@ def _to_intermediate_stack(
                         )
                         if (
                             inst.opname in _UNARY_OPCODES
-                            or inst.opname in _SYNTHETIC_OPS
+                            or _SYNTHETIC_OPS.get(inst.opname) == 1
```

Comment:
is the idea to then later add more `_SYNTHETIC_OPS` with a different value?

Reply:
Yup - I had some potential ideas where we might want to inject a synthetic binary op, in which case it would be registered with a value of two (eg: 1 = unary, 2 = binary).